agentops/llm_tracker.py

```diff
@@ -59,7 +78,7 @@ def handle_stream_chunk(chunk):
                     self.client.record(self.event_stream)
                     self.event_stream = None
             except Exception as e:
-                print(
+                logging.info(
```

Comment:
I have all of the my logs start with "AgentOps:". I think we should try to be consistent. Also, I think this deserves at least a logging.warning().

Reply:
bet

Reply:
I didn't write the log, just switched it from print, but I can add it